JitsiMeetJS.js

```diff
@@ -340,6 +340,17 @@ export default _mergeNamespaceAndModule({
                                 () => {
                                     Statistics.stopLocalStats(mStream);
                                 });
+                            track.addEventListener(
```

Comment:
Maybe this could be embedded in statistics since `startLocalStats`, since it gets a reference to the stream? Maybe we could  pass the track instead. Then I think you'd avoid having to create new public APIs which are a bit weird on their own.

Reply:
done, also moved the existing eventlistener inside `startLocalStats`.